lib/main.js

```diff
@@ -49,20 +49,27 @@ function errorHandler(err) {
 async function runElmReview() {
   const {elmModulePath, reviewElmJson, appHash} = await Builder.build(options);
 
-  if (!elmModulePath) {
+  if (!elmModulePath || !options.elmJsonPath) {
     AppState.exitRequested(1);
     return;
   }
 
+  const optionsWithElmJsonPath = {...options, elmJsonPath: options.elmJsonPath};
```

Comment:
For now, we just exit if elmJsonPath is null. We could have better errors if you want (if so, we could wrap this in a function and just call it where needed) 🙂